cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/FlowProgramRunner.java

```diff
@@ -251,6 +251,8 @@ protected void doCommand(String name, Object value) throws Exception {
         changeInstances(command.get("flowlet"), Integer.valueOf(command.get("newInstances")));
       } catch (Throwable t) {
         LOG.error(String.format("Fail to change instances: %s", command), t);
+        stop();
```

Comment:
What will this stop do ? 
If a flow is running and someone tries to change the instance which for some reason fails then it will stop the running flow ?